### ISSUE TITLE
is this your classmate review?

### ISSUE THREAD

**Issue by fedhere** [zem232/PUI2018_zem232#3]
is this your classmate revoew? PUI2018_zem232/HW4_zem232/.ipynb_checkpoints/HW4_zem232_citibikes-checkpoint.ipynb

please move it to the HW4_zem232 file. It should not be in a hidden file (.ipynb_checkpoints) - it is not your fault, it is how they submitted the pull request, but please fix it

**Comment by zem232** [comment]
Yes that was the review that shy256 left for me. How do I fix it? Should I just download the notebook and re-upload as a new notebook, or is there an easier way to fix it?
